src/main/java/bio/terra/service/filedata/azure/util/AzureBlobStoreBufferedWriter.java

```diff
@@ -0,0 +1,20 @@
+package bio.terra.service.filedata.azure.util;
+
+import com.azure.storage.blob.BlobClientBuilder;
+import java.io.BufferedWriter;
+import java.io.OutputStreamWriter;
+import java.nio.charset.StandardCharsets;
+
+/** Given a url to an azure blob, return a buffered reader for the blob */
+public class AzureBlobStoreBufferedWriter extends BufferedWriter {
+  public AzureBlobStoreBufferedWriter(String ingestRequestSignURL) {
```

Comment:
Wanted to point to this in case it's helpful: Here's how we're writing to an azure blob in our tests: https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/test/java/bio/terra/service/filedata/azure/util/BlobIOTestUtility.java#L150

Reply:
The test utility method uploads a byte stream vs a stream of strings. There's not really a great way to transform a stream of strings to a stream of bytes in an efficient way, so I think this is the way we'll want to do it in prod.